packages/perseus/src/widgets/radio/util.ts

```diff
@@ -20,16 +27,22 @@ export function getChoiceLetter(pos: number, strings: PerseusStrings): string {
     return " ";
 }
 
+/**
+ * @deprecated and likely a very broken API
+ * [LEMS-3185] do not trust serializedState/restoreSerializedState
+ */
 export function getUserInputFromSerializedState(
-    props: Props,
+    serializedState: any,
```

Comment:
Serialized state isn't technically props: it's the history of every type of RenderProps we've ever had. That's why it's so dangerous.